src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -187,9 +184,6 @@ let PostThreadItemLoaded = ({
     return makeProfileLink(post.author, 'post', urip.rkey, 'reposted-by')
   }, [post.uri, post.author])
   const repostsTitle = _(msg`Reposts of this post`)
-  const isModeratedPost =
-    moderation.decisions.post.cause?.type === 'label' &&
-    moderation.decisions.post.cause.label.src !== currentAccount?.did
 
   const translatorUrl = getTranslatorLink(
     record?.text || '',
```

Comment:
can we use "??" instead of "||"

 const translatorUrl = getTranslatorLink(
    record?.text ?? ""

Reply:
I Guess that in this case, It's same behavior with ??.